code/src/java/pcgen/cdom/enumeration/StringKey.java

```diff
@@ -33,16 +33,16 @@
  */
 public enum StringKey
 {
-	NAME, KEY_NAME, WORSHIPPERS, TITLE, HOLY_ITEM, MASTER_HP_FORMULA, MASTER_CHECK_FORMULA, 
+	NAME, KEY_NAME, MASTER_HP_FORMULA, MASTER_CHECK_FORMULA, 
 	MASTER_BAB_FORMULA, BIRTHPLACE, BIRTHDAY, CHOICE_STRING, DESTINATION, 
-	PAGE_USAGE, MINDEVVER, MINVER, BIO, DESCRIPTION, GENRE, HELP, APPEARANCE, CATCH_PHRASE, 
+	PAGE_USAGE, MINDEVVER, MINVER, BIO, DESCRIPTION, GENRE, HELP, CATCH_PHRASE, 
 	OUTPUT_NAME, PUB_NAME_LONG, COST, PUB_NAME_SHORT, 
 	CURRENT_EQUIP_SET_NAME, PUB_NAME_WEB, SETTING, TEMP_DESCRIPTION, TRAIT2, 
 	TRAIT1, SKIN_COLOR, HAIR_COLOR, SPEECH_TENDENCY, PHOBIAS, INTERESTS, RESIDENCE, 
 	SPELLBOOK_AUTO_ADD_KNOWN, PLAYERS_NAME, HANDED, HAIR_STYLE, PORTRAIT_PATH, EYE_COLOR, LOCATION, 
-	FILE_NAME, ABB, TARGET_AREA, NAME_TEXT, DAMAGE, DAMAGE_OVERRIDE, RATE_OF_FIRE, FUMBLE_RANGE, 
-	LISTTYPE, SPELLTYPE, ITEMCREATE, LEVEL_TYPE, DATA_PRODUCER, DATA_FORMAT, CAMPAIGN_SETTING,
+	FILE_NAME, ABB_KR, TARGET_AREA, NAME_TEXT, DAMAGE, DAMAGE_OVERRIDE, FUMBLE_RANGE, 
```

Comment:
There were two versions of ABB in the code, ABB for classes (what I temporarily called ABB_CL but was never seen in the external world) and ABB where it was a key replacement (what I now call ABB_KR).  So it's what was ABB on Size, Alignment, Stats